packages/next/pages/_app.js

```diff
@@ -29,11 +27,13 @@ export default class App extends Component {
   }
 
   render () {
-    const {router, Component, pageProps} = this.props
+    const {router, Component, pageProps, headManager} = this.props
     const url = createUrl(router)
-    return <Container>
-      <Component {...pageProps} url={url} />
-    </Container>
+    return <HeadManagerContext.Provider value={headManager}>
```

Comment:
This should be added in `packages/next/client/index.js`, as users override _app.js with their own render method.